includes/derivatives.inc

```diff
@@ -159,7 +159,7 @@ function islandora_large_image_create_jpg_derivative(AbstractObject $object, $fo
 
     $uploaded_file = islandora_large_image_get_uploaded_file($object);
     $args = array();
-    $args[] = '-resize ' . escapeshellarg("600 x 800");
+    $args[] = '-resize ' . escapeshellarg(variable_get('derivative_med_size', "600 x 800"));
```

Comment:
Is your intent to make a re-usable variable across multiple modules that get used here?